x/ecocredit/client/tx.go

```diff
@@ -98,7 +98,7 @@ const (
 
 func TxGenBatchJSONCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "gen-batch-json [--class-id class_id] [--issuances issuances] [--start-date start_date] [--end-date end_date] [--project-location project_location] [--metadata metadata]",
+		Use:   "gen-batch-json --class-id [class_id] --issuances [issuances] --start-date [start_date] --end-date [end_date] --project-location [project_location] --metadata [metadata]",
```

Comment:
It looks like the `Long` description is outdated. `issuance` should be `issuances`, `project_location` is missing, `issuer` is no longer needed. Should we still be referring to them as parameters? Maybe required flags?

Reply:
ah good catch! i updated the descriptions, and made the order they appear reflect the usage string order as well. thanks!!

https://github.com/regen-network/regen-ledger/pull/472/commits/3a1064cdaf25e93434bc63be1ac92588d695ca70